frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -40,6 +41,10 @@ common_settings.screen = {
         require("ui/elements/refresh_menu_table"),
     },
 }
+if Device:isAndroid() then
+    table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
```

Comment:
@chrox so now fix is in master, right?
So as I did `git merge upstream/master` it should work on my ubuntu 16.04, right?
Or should I do anything more as it's again freezed after install?